core/services/form/meta/inputs/Attributes.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace EventEspresso\core\services\form\meta\inputs;
+
+class Attributes
+{
+
+}
```

Comment:
Missing new line.

Reply:
oops.. meant to actually remove this class for now, thnx for bringing it to my attention